backend/src/helpers/config.js

```diff
@@ -27,7 +27,14 @@ const fastifyAuth = require('fastify-auth');
             const token = getBearerTokenFromRequest(req);
             if (token) {
                 verifyToken(token, jwksUri)
-                    .then(() => done())
+                    .then((res) => {
+                        req.log.info(res);
```

Comment:
This logging gives feedback when a user successfully authenticates, and is either added to the DB or not. We don't need to log for these things. But depending on how we're using logging(if it's to debug), then this will print something when a user logs in, as opposed to nothing(unless there's an error).